doc/user_guide/user_guide.md

```diff
@@ -82,23 +82,59 @@ cat language_container_part_* > language_container.tar.gz
 ```
 
 #### Install Language Container
-- To install the language container, it is necessary to load the container into the BucketFS 
-and register it to the database. The following command provides this setup:
-```buildoutcfg
```

Comment:
"buildoutcfg" seems to be not in list of [markdown_supported_languages](https://github.com/jincheng9/markdown_supported_languages). Maybe you want to replace it by "shell"?